integration-tests/__tests__/to_match_snapshot.test.js

```diff
@@ -159,3 +159,65 @@ test('accepts custom snapshot name', () => {
     expect(status).toBe(0);
   }
 });
+
+test('handles property matchers', () => {
+  const filename = 'handle-property-matchers.test.js';
+  const template = makeTemplate(`test('handles property matchers', () => {
+      expect({createdAt: $1}).toMatchSnapshot({createdAt: expect.any(Date)});
+    });
+    `);
+
+  {
```

Comment:
weird to use scopes here, although I get why you do it. why not just a `descirbe` around it and multiple tests instead?

Reply:
My thoughts as well - I was copying the style from the rest of the suite. I'll update the suite 👌